### DISCUSSION TITLE
Add libsql:// connection for sqlite 🫧

### DISCUSSION THREAD

**Discussion by petcard** [adonisjs/core#4486] [Ideas]
Hey 👋

It will be really nice if the sqlite connection can handle [Turso](https://turso.tech/) database like this `libsql://adonis.turso.io`.
Sqlite is getting more and more popular and we can already use sqlite with Adonis.

Thank you, love the work you are all doing 🫶

**Comment by neupauer** [comment]
Hi, check out this discussion https://github.com/orgs/adonisjs/discussions/4299#discussioncomment-11377065
